modules/bending_energy_diffusion_loss_notes.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<p>unimportant phrasing suggestion here and below:</p><ul><li>shall -&gt; should</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1589/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>